src/Console/Command/Qa/QaStaticAnalysisCommand.php

```diff
@@ -148,7 +148,7 @@ protected function configure():void {
       ->setHelp('Tools can be specified individually or in combination. If none are specified, all will be run.')
       ->addArgument('path', InputArgument::REQUIRED, 'The path to analyze')
       ->addOption('composer', NULL, InputOption::VALUE_NONE, 'Run the Composer validation tool')
-      ->addOption('coverage', NULL, InputOption::VALUE_NONE, 'Run the code coverage estimator. Implies "--phploc"')
+      ->addOption('coverage', NULL, InputOption::VALUE_NONE, 'Run the code coverage estimator.')
```

Comment:
Ooh... I forgot about this. 😬 The health score calculator depends on PHPLOC data. Okay, we need to take this to the stakeholders.

Reply:
This change should be reverted. It's accurate again now that PHPLOC is remaining.